pkg/apiserver/api_installer.go

```diff
@@ -105,7 +106,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 
 	serverVersion := a.group.ServerVersion
 	if len(serverVersion) == 0 {
-		serverVersion = a.group.Version
+		serverVersion = apiutil.GetGroupVersion(a.group.Group, a.group.Version)
```

Comment:
if server version is supposed to be a group version, then why do you need this if? Just compute it directly.